DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -78,35 +79,113 @@
             </li>
             <li class="nhsuk-header__navigation-item">
-              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/learningportal/help/LearningPortal.html" target="_blank">
-                Help
+              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/Home#products">
+                Products
                 <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
                   <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
                 </svg>
               </a>
             </li>
             <li class="nhsuk-header__navigation-item">
-              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/logout">
-                Log out
+              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/Home#learningcontent">
+                Learning content
                 <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
                   <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
                 </svg>
               </a>
             </li>
+            <li class="nhsuk-header__navigation-item">
+              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/findyourcentre">
+                Find your centre
+                <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                  <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                </svg>
+              </a>
+            </li>
+            <li class="nhsuk-header__navigation-item">
+              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/pricing">
+                Pricing
+                <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                  <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                </svg>
+              </a>
+            </li>
+            @{
+              string helpPage;
+              if (User.HasClaim(CustomClaimTypes.UserUserAdmin, "True")) {
+                helpPage = "/help/Introduction.html";
+              } else if (User.Identity.IsAuthenticated) {
+                helpPage = "/learningportal/help/LearningPortal.html";
+              } else {
+                helpPage = "/help/Introduction.html";
+              }
+            }

```

Comment:
I'm not sure if there's a more idiomatic way of setting this, my C# is a bit rusty.

Reply:
I think this is the best way we have. Might be able to check if the UserUserAdmin is False and display the learningportal.html, otherwise should introduction, but I am going to need to double check that the log in system actually works such that that would be feasible.

Reply:
I've done some extra digging on this - it looks like the only guaranteed way to check if a user is an admin is to check if the User has a claim of CustomClaimTypes.UserAdminId - if the user is NOT an admin then this claim will never be added. 

Other than that, I think the code is fine. I don't think there is a way to make those checks any better due to the way that the log in system mashes the different claims together if the user has both admin and delegate accounts.

Reply:
Checking for admin id sounds good - regarding the if/else if/else - I think the logic is good as is, because HEE will come back to us with a "not logged in" resource at some point and it will make the change easy.